src/misc/lv_event.h

```diff
@@ -111,9 +111,14 @@ typedef enum {
 
     LV_EVENT_PREPROCESS = 0x8000,   /** This is a flag that can be set with an event so it's processed
                                       before the class default event processing */
+    LV_EVENT_MARKED_DELETING = 0x10000,
 } lv_event_code_t;
 
-typedef lv_array_t lv_event_list_t;
+typedef struct {
+    lv_array_t array;
+    bool is_traversing;
```

Comment:
It is more concise to use a flags. If other flags exist, a bool may be added.

Reply:
At present, there is no other requirement for `flag`, and it is more concise to use bool.

The demand for other `flags` has not been increased in a short period of time.